.gitlab-ci.yml

```diff
@@ -367,6 +367,7 @@ test-prometheus-alerting-rules:
     - curl -L https://github.com/prometheus/prometheus/releases/download/v2.19.0/prometheus-2.19.0.linux-amd64.tar.gz --output prometheus.tar.gz
     - tar -xzf prometheus.tar.gz
     - ./prometheus-*/promtool check rules .maintain/monitoring/alerting-rules/alerting-rules.yaml
```

Comment:
I created a pr for adding promtool directly to the tools image: https://github.com/paritytech/scripts/pull/209